src/main/java/fi/helsinki/cs/tmc/cli/Application.java

```diff
@@ -176,6 +172,20 @@ public void run(String[] args) {
     }
 
     public static void main(String[] args) {
+        while(true) {
```

Comment:
Do we want to have your debugging loop as the main entry point for all of our users?

Reply:
:dolphin: